new-examples/bbenavswanlake.yml

```diff
@@ -341,9 +341,6 @@
     - title: Transactional Qualifier 
       url: /learn/by-example/transactional-qualifier
       active: transactional-qualifier
-    - title: Distributed Transactions 
```

Comment:
@hasithaa @praneesha removed distributed transactions BBE as we dont have a BBE for distributed case for beta1